src/components/boardComponent/BoardComponent.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import GameBoard from './boardComponent';
+import Square from '../squareComponent/squareComponent';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Board Component', () => {
+  it('renders 9 Square components', () => {
+    const wrapper = mount(<GameBoard />);
+
+    expect(wrapper.find(Square).length).toBe(9);
+  });
+
+  describe('Value of Marker', () => {
+    it('is an O', () => {
+      const wrapper = mount(<GameBoard />);
+
+      wrapper.find(Square).first().simulate('click');
+
+      expect(wrapper.state('value')).toBe('O');
+    });
+
+    it('is a X', () => {
+      const wrapper = mount(<GameBoard />);
+
+      wrapper.find(Square).at(0).simulate('click');
+      wrapper.find(Square).at(1).simulate('click');
+
+      expect(wrapper.state('value')).toBe('X');
+    });
+  });
+
+
+  describe('Duplicate clicks', () => {
+    it('does not change the value', () => {
+      const wrapper = mount(<GameBoard />);
+
+      wrapper.find(Square).at(0).simulate('click');
+      wrapper.find(Square).at(0).simulate('click');
+
+      expect(wrapper.state('value')).toBe('X');
+    });
+  });
+
+  describe('Value placed in the square', () => {
```

Comment:
This `describe` and the `Value of marker` above are very similar to me.

Reply:
I would try to test 2 things: 1. That the correct marker is placed on the board. and 2. They alternate between X and O (looks like the board is the only thing controlling that for now). It seems to me these two `describe`s both kinda test both of those things. So they are repeatable and yet not clear what is being tested. What do you think? @akiraandy